### PR TITLE
update argument descriptions in notebook and docstrings; significantly, GetPopulations constraintsPVMap required->optional with NULL default

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: notebooks/demo-notebook.Rmd [hunk 1]
```diff
@@ -249,12 +250,12 @@ The final property of interest is **`measuredValue`**. This property tells us th
...
-- **`measuredProperty`** - The `measuredProperty` of the observation.
-- **`statsType`** - The statistical type of the observation.
-- **`observationDate`** - The `observationDate` of the observation.
-- **`observationPeriod`**`[=NULL]` - An optional parameter specifying the `observationPeriod` of the observation.
-- **`measurementMethod`**`[=NULL]` - An optional parameter specifying the `measurementMethod` of the observation.
+- **`dcids`** - vector OR single column tibble/data frame of dcids to get observations for.
+- **`measuredProperty`** - string identifying the measured property.
+- **`statsType`** - string identifying the statistical type for the observation.
+- **`observationDate`** - string of the associated observation date in ISO8601 format.
+- **`observationPeriod`** - optional string specifying the observation period. NULL by default.
```
Comment (beets) [inline notebooks/demo-notebook.Rmd]: These are not truly optional in the sense that, if they are specified in the observation, they must be specified.

File: notebooks/demo-notebook.Rmd [hunk 2]
```diff
@@ -119,11 +119,12 @@ To see the documentation for any function, you can bring up the R doc in the Hel
...
 
-- **`dcids`** - A vector OR single-column tibble/data frame of string(s) that identify node(s) to get property values for.
-- **`prop`** - The property to get property values for.
-- **`outgoing`**`[=TRUE]` - An optional flag that indicates the property is oriented away from the given nodes if true.
-- **`valueType`**`[=NULL]` - An optional parameter which filters property values by the given type. 
-- **`limit`**`[=100]` - An optional parameter which limits the total number of property values returned *aggregated over all given nodes*.
+- **`dcids`** - vector OR single-column tibble/data frame of string(s) of dcid(s) to get property values for.
+- **`prop`** - string identifying the property to get the property values for.
+- **`outgoing`** - optional boolean flag indicating whether the property
+#'   is directed away from the given nodes. TRUE by default
```
[the remark this answers is not in the document]
Comment (tjann) [reply notebooks/demo-notebook.Rmd]: No, thanks for the catch!

File: notebooks/demo-notebook.Rmd (hunk 1 above)
Comment (tjann) [reply notebooks/demo-notebook.Rmd]: added this in f18ffa

File: notebooks/demo-notebook.Rmd [hunk 3]
```diff
@@ -211,9 +212,9 @@ There are two other properties defined: `numConstraints` and `employment`. These
 
 To query for `StatisticalPopulation`s using the Data Commons R API Client, we call **`GetPopulations`**. The function accepts the following parameters.
 
-- **`dcids`** - A list or Pandas Series of dcids denoting the locations of populations to query for.
-- **`populationType`** - The `populationType` of the `StatisticalPopulation`
-- **`constraintsPVMap`** - An optional map from constraining property to the value that the `StatisticalPopulation` should be constrained by.
+- **`dcids`** - vector OR single-column tibble/dataframe of `dcid`(s) identifying `Place`s of `StatisticalPopulation`s to query for.
```
[the remark this answers is not in the document]
Comment (tjann) [reply notebooks/demo-notebook.Rmd]: code-faced dcid

How about we do that with the big face-lift once everything is finalized? For now I'm tracking Python readthedocs

[the remark this answers is not in the document]
Comment (beets) [reply notebooks/demo-notebook.Rmd]: sgtm

File: datacommons/R/popobs.R [hunk 4]
```diff
@@ -75,9 +74,13 @@
 #'                               list(gender = 'Male'))
 #' df$femalePops <- GetPopulations(select(df, countyDcid), 'Person',
 #'                                 list(gender = 'Female'))
-GetPopulations <- function(dcids, populationType, constraintsPVMap) {
+GetPopulations <- function(dcids, populationType, constraintsPVMap = NULL) {
```
Comment (beets) [inline datacommons/R/popobs.R]: for cleanliness, can you mention this in the PR description?

**Pull request merged**